.eslintrc.js

```diff
@@ -6,11 +6,13 @@ module.exports = {
     sourceType: 'module',
   },
   plugins: [
-    'ember'
+    'ember',
+    'prettier',
   ],
   extends: [
     'eslint:recommended',
     'plugin:ember/recommended',
+    'prettier',
```

Comment:
If you tweak this to `plugin:prettier/recommended` (instead of `prettier`), you do not need to add `prettier/prettier` to rules below.

Reply:
Oooh, neat, I'll push that.